bpf/lib/nodeport.h

```diff
@@ -1260,6 +1260,12 @@ static __always_inline int nodeport_svc_lb6(struct __ctx_buff *ctx,
 	ret = lb6_local(get_ct_map6(tuple), ctx, l3_off, l4_off,
 			key, tuple, svc, &ct_state_svc,
 			nodeport_xlate6(svc, tuple), ext_err, 0);
+#ifdef ENABLE_L7_LB
+	if (ret == DROP_PUNT_PROXY) {
```

Comment:
Similar to `NAT_PUNT_TO_STACK`, would it make sense to introduce a `LB_PUNT_TO_STACK` alias?

Reply:
Makes sense, will add an alias & repush, thx!